backend/src/extension.ts

```diff
@@ -123,25 +123,29 @@ export class YeomanUIPanel {
 						const resolve = this.questionsResolutions.get(message.taskId);
 						resolve(message.data);
 						return;
+					case 'showDoneMessage':
+						let Close = 'Close';
+						let OpenWorkspace = 'Open Workspace';
+						this.theia.isInTheia().then((value) => {
+							let commandName_Close = "workbench.action.closeActiveEditor";
+							let commandName_OpenWorkspace = "vscode.openFolder";
+							let commandParam = _.get(message, "commandParams[0]");
+							vscode.window.showInformationMessage('Where would you like to open the project?', Close , OpenWorkspace)
+								.then(selection => {
+									if (selection === Close) {
+										this.executeCommand(commandName_Close, undefined);
+									}
+									if (selection === OpenWorkspace) {
```

Comment:
change to else if
selection cannot be Close and OpenWorkspace at the same time

Reply:
done